doc/getting-started/installation.rst

```diff
@@ -163,7 +163,7 @@ See other examples and demos:
     see our `Contributing Guide`_ and specifically, the `Testing`_ section.
 
 .. _Contributing Guide: https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst
-.. _Testing: https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst#testing
+.. _Testing: https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst#user-content-testing
```

Comment:
Now this is also weird: #testing seems to be a valid anchor on the same page.

Reply:
I'll test them all at once again in a moment :)

Reply:
1) ` https://github.com/googlefonts/noto-emoji#license - Anchor 'license' not found `

      Replaced by: https://github.com/googlefonts/noto-emoji/blob/main/LICENSE

2) `broken    https://github.com/InsightSoftwareConsortium/itkwidgets#installation - Anchor 'installation' not found
`
     Replaced by: https://itkwidgets.readthedocs.io/en/latest/quick_start_guide.html

3) `broken https://www.researchgate.net/  403 Client Error`

     So we will ignore, oki

4) `broken    https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst#testing - Anchor 'testing' not found`
     Replaced by:  https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst#user-content-testing

Reply:
Ah, OK, looking at `rewrite_github_anchor()` at https://www.sphinx-doc.org/en/master/_modules/sphinx/builders/linkcheck.html now I realise that your change only fixes the anchor while linking to the same item. So you exactly did the fix that linkcheck should be doing, so again this is fine. Sorry for being slow on the uptake.

Reply:
no worries, it took me some time to figure this out and didn't work for all. but fun, learning! :)